python/lsst/pipe/tasks/processBrightStars.py

```diff
@@ -211,31 +192,128 @@ class ProcessBrightStarsTask(PipelineTask):
 
     def __init__(self, initInputs=None, *args, **kwargs):
         super().__init__(*args, **kwargs)
-        # Compute (model) stamp size depending on provided "buffer" value
-        self.modelStampSize = [
-            int(self.config.stampSize[0] * self.config.modelStampBuffer),
-            int(self.config.stampSize[1] * self.config.modelStampBuffer),
+        self.setModelStamp()
+
+    def runQuantum(self, butlerQC, inputRefs, outputRefs):
+        self.config = cast(ProcessBrightStarsConfig, self.config)
```

Comment:
Again, not a fan of seemingly unnecessary casts, unless it has to be done to appease mypy.

Reply:
All casts, here and elsewhere, removed.